message DseEquipmentRefine
{
	optional string		oid = 1;	//装备唯一ID
	optional int32		code = 2;	//错误码
}
